app/views/calculators/take_whole_pot/_calculator.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render 'form', calculator: @calculator, pot: @pot, income: @income %>
```

Comment:
You should use the locale variables passed to the partial not the instance variables defined by the controller. This allows the partial to be used in any context without it needing to know how it is being used.

```rb
<%= render 'form', calculator: calculator, pot: pot, income: income %>
<%= render 'result', calculator: calculator %>
```